code/src/components/Artist.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+
+ // Creating a new component
+const Artist = (props) => {
+   return (
+
+      // <div className="artist">
+        <a className="linked-artist" href={props.artistLink} target="_blank" rel="noopener noreferrer"> 
+        {props.artistName}
+        </a>
+         // </div>
+
```

Comment:
Neat as everything else. Feels like you've aimed at keeping it as short as possible - good job.